### PR TITLE
doc/release-notes: ensure RBD users can blacklist prior to upgrade

### PR DESCRIPTION
**Pull request by dillaman**
Fixes: http://tracker.ceph.com/issues/21353
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

### CODE REVIEW

File: doc/release-notes.rst [hunk 1]
```diff
@@ -163,7 +163,8 @@ Major Changes from Kraken
   * Specifying user authorization capabilities for RBD clients has been
     simplified. The general syntax for using RBD capability profiles is
     "mon 'profile rbd' osd 'profile rbd[-read-only][ pool={pool-name}[, ...]]'".
-    For more details see "User Management" in the documentation.
+    For more details see :doc:`rados/operations/user-management/`
```
Comment (amitkumar50) [inline doc/release-notes.rst]: rados/operation/user-management does not looks to be directory.
this is file user-management.rst

File: doc/release-notes.rst [hunk 2]
```diff
@@ -417,6 +418,12 @@ Upgrade from Jewel or Kraken
 
      # ceph osd set noout
 
+#. Verify that all RBD client users have sufficient caps to blacklist
+   other client users. RBD client users with only ``"allow r"``
+   monitor caps shold to be updated as follows::
```
Comment (amitkumar50) [inline doc/release-notes.rst]: shold spelling wrong

**Comment by smithfarm** [comment]
@theanalyst Do we need to include this in the v12.2.1 release notes?

@dillaman Can you correct the spelling of "should" as per @amitkumar50 review?

**Comment by dillaman** [comment]
I am out of the country presently -- if it can wait until next week, I'll fix it. Otherwise, can someone else open a new PR w/ the spelling fix? Thanks!

**Comment by theanalyst** [comment]
@smithfarm since this is a doc only change lets just wait until we have a decision on whether we are going for a new build, if not I'll just merge in after we do a release

File: doc/release-notes.rst (hunk 1 above)
Comment (dillaman) [reply doc/release-notes.rst]: The ":doc:" tag used throughout this file just links to the directories that are created when the ```.rst``` files are processed.

**Comment by dillaman** [comment]
spelling correction pushed

**Pull request merged**